default_shardings/gemma.yaml

```diff
@@ -3,7 +3,9 @@
 # "replicated" to signify "replicated".
 # Integer signify axis to shard: 0 <= shard axis < rank
 
-freqs_cis : -1 #  torch.complex64 (16384, 128)
+freqs_cis : null #  torch.complex64 (16384, 128)
+layers.*.self_attn.qkv_proj.weight: 0
```

Comment:
Is this one only for test purpose? In gemma model, I saw the code directly read wq,wk and wv.